### DISCUSSION TITLE
Implementing "Material.Avalonia" themes at run time.

### DISCUSSION THREAD

**Discussion by CarlosEX** [AvaloniaUI/Avalonia#5623] [General]
Hi guys, gradually I'm managing to implement the themes of Material.Avalonia. There's still a lot left to improve the codes, because i'm actually newbie yet. But I already like the result. This project is still at the beginning and other projects, I have already put the videos on my personal website.

[Site](https://carlosantoniocison.editorx.io/portifolio)

![avaloniaui_carlosantonio](https://user-images.githubusercontent.com/33513870/110203063-bdaf6400-7e4a-11eb-9f29-7559a92376f8.gif)

My first project avalonia me, using the citrus theme. This application was created to consume the IBM Watson TTS service.
[Avalonia Theme Citrus](carlosantoniocison.editorx.io/portifolio)

**Comment by danwalmsley** [comment]
Muito Bom! Well done! this is amazing!

**Reply by danwalmsley** [reply]
Can we tweet your video?

**Reply by CarlosEX** [reply]
It would be an honor for me. The full video is this [video](https://youtu.be/BRWTVAwclCU). O project still has a lot to do, and I'm now beginning to understand how XAML and Avalonia progração works. It's only been a month since I started working with it.

**Reply by CarlosEX** [reply]
Yes

**Comment by robloo** [comment]
This is really cool!

Long-term it would be great to see some equivalent themes for Android (material) and iOS/macOS to find their way into Avalonia like Fluent did. I know there is concern about maintainability but having native-looking themes for each platform would be extremely useful.
